packages/app/src/vis-packs/ValueLoader.module.css

```diff
@@ -6,8 +6,34 @@
   flex-direction: column;
 }
 
+.progressBars {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 20rem;
+  margin-bottom: 0.75rem;
```

Comment:
I would change the CSS (add a `min-height` ?) so that the `Cancel` button is not moved around when progress bars disappear. 

It might be frustrating for the user to try to click on the cancel button if the layout changes dynamically...

Reply:
Good call. I've added a `min-height` as you suggested, as well as `justify-content: center` so the whitespace is shared equally above/below the bar(s) when there's only one or two.